scripts/build_llvm.sh

```diff
@@ -91,22 +91,29 @@ if [ "$(echo ${projects[*]} | xargs)" != "" ]; then
   unset llvm_components
   install_target=install
 else 
-  install_target=install-distribution-stripped
+  install_target=install # Fixme: should be install-distribution-stripped
+  # some possibly useful components or python bindings:
+  # libclang-python-bindings, MLIRPythonModules, core-resource-headers;clang-resource-headers
 fi
 
+# A hack, since otherwise the build can fail due to line endings in the LLVM script:
+cat "../llvm/cmake/config.guess" | tr -d '\r' > ~config.guess
+cat ~config.guess > "../llvm/cmake/config.guess" && rm ~config.guess
+
 # Generate CMake files
 cmake_args="-G Ninja ../llvm \
   -DLLVM_TARGETS_TO_BUILD="host" \
   -DCMAKE_BUILD_TYPE=$build_configuration \
   -DCMAKE_INSTALL_PREFIX="$LLVM_INSTALL_PREFIX" \
   -DLLVM_ENABLE_PROJECTS="$llvm_projects" \
   -DLLVM_DISTRIBUTION_COMPONENTS=$llvm_components \
+  -DLLVM_ENABLE_BINDINGS=OFF \
+  -DMLIR_ENABLE_BINDINGS_PYTHON=TRUE \
```

Comment:
If the build allows, can we use 'ON/OFF' for sake of consistency and readability? 
```suggestion
  -DMLIR_ENABLE_BINDINGS_PYTHON=ON \
```